wpeverest/user-registration
# Refactor- popup UI

### All Submissions:






### Changes proposed in this Pull Request:



This PR is to refactor the design consistency for popup UI for admin panel.

**Before refactor:**
![image](https://user-images.githubusercontent.com/31344435/98773922-03ff3400-2412-11eb-903b-ff044e23b4bd.png)

**After Refactor:**
![image](https://user-images.githubusercontent.com/31344435/98773947-0f525f80-2412-11eb-941a-36d74901edb4.png)


### How to test the changes in this Pull Request:

1. Checkout to the branch.
2. Check the popup designs.

### Types of changes:




### Other information:




### Changelog entry

> Refactor - Popup UI
